xbmc/filesystem/File.h

```diff
@@ -110,6 +111,7 @@ class CFile
   void Close();
   int GetChunkSize();
   const std::string GetProperty(XFILE::FileProperty type, const std::string &name = "") const;
+  const std::vector<std::string> GetProperties(XFILE::FileProperty type, const std::string &name = "") const;
```

Comment:
`... const override;`

Reply:
Its not an override here. The compiler complains if I add "override".

Reply:
you are right, override is missing in a different file, which overrides this function